server/src/main/java/org/opensearch/discovery/InitializeExtensionsRequest.java

```diff
@@ -56,21 +61,31 @@ public DiscoveryNode getSourceNode() {
         return sourceNode;
     }
 
+    public int getPort() {
+        return port;
+    }
+
+    public void setPort(int port) {
+        this.port = port;
+    }
+
     @Override
     public String toString() {
-        return "InitializeExtensionsRequest{" + "sourceNode=" + sourceNode + ", extensions=" + extensions + '}';
+        return "InitializeExtensionsRequest{" + "sourceNode=" + sourceNode + ", extensions=" + extensions + ", port=" + port + '}';
```

Comment:
(Suggestion) might be more readable to have the port immediately following the node, rather than after a (possibly long) list of extensions.

Reply:
Sure. Will take care of it in the next PR.